### PR TITLE
Release 133.0.0

### PR DESCRIPTION
**Pull request by mcmire**
This release includes:

- `@metamask/announcement-controller` 6.0.1 -> 6.1.0
- `@metamask/gas-fee-controller` 14.0.1 -> 15.0.0
- `@metamask/network-controller` 18.0.1 -> 18.1.0
- `@metamask/queued-request-controller` 0.6.1 -> 0.7.0
- `@metamask/selected-network-controller` 10.0.1 -> 11.0.0
- `@metamask/transaction-controller` 25.3.0 -> 26.0.0
- `@metamask/user-operation-controller` 6.0.0 -> 7.0.0

### CODE REVIEW

File: packages/announcement-controller/CHANGELOG.md [hunk 1]
```diff
@@ -123,7 +129,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
     All changes listed after this point were applied to this package following the monorepo conversion.
 
-[Unreleased]: https://github.com/MetaMask/core/compare/@metamask/announcement-controller@6.0.1...HEAD
+[Unreleased]: https://github.com/MetaMask/core/compare/@metamask/announcement-controller@6.1.0...HEAD
+[6.1.0]: https://github.com/MetaMask/core/compare/@metamask/announcement-controller@6.0.0...@metamask/announcement-controller@6.1.0
```
Comment (digiwand) [inline packages/announcement-controller/CHANGELOG.md:133]:
```suggestion
[6.1.0]: https://github.com/MetaMask/core/compare/@metamask/announcement-controller@6.0.1...@metamask/announcement-controller@6.1.0
```

Comment (mcmire) [reply packages/announcement-controller/CHANGELOG.md]: Thanks! Fixed in [8d42e5d](https://github.com/MetaMask/core/pull/4121/commits/8d42e5da1db329e2bc43affffd9f243b3db50a83).

File: packages/gas-fee-controller/CHANGELOG.md [hunk 2]
```diff
@@ -7,6 +7,17 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+
+### Changed
+
+- **BREAKING**: The controller's constructor now requires `infuraAPIKey`. This is used to construct and send the `Authorization` header for Infura gas API requests. ([#4068](https://github.com/MetaMask/core/pull/4068))
+- Bump dependency `@metamask/network-controller` to `^18.1.0` ([#4121](https://github.com/MetaMask/core/pull/4121))
+
+### Removed
+
+- **BREAKING**: Remove the constructor options `legacyAPIEndpoint` and `EIP1559APIEndpoint`. These URLs are now hardcoded within the controller. ([#4068](https://github.com/MetaMask/core/pull/4068))
+
```
Comment (digiwand) [inline packages/gas-fee-controller/CHANGELOG.md:20]: 👍🏼

**Review by adonesky1 — APPROVED** [review]
LGTM!

**Review by digiwand — APPROVED** [review]
nice updates and added changelogs 👍🏼 

e.g. despite not being shown in the related PR description
```
- Add `getLayer1GasFee` method to `TransactionController` to get the layer 1 gas fee for the given transaction params ([#4055](https://github.com/MetaMask/core/pull/4055))
```

**Review by Gudahtt — APPROVED** [review]
LGTM!

**Pull request merged**